packages/gasket-plugin-express/generator/routes/index.js

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable no-unused-vars */
+require('dotenv').config();
```

Comment:
I really think this should happen earlier, as in the first line of the Gasket CLI [here](https://github.com/godaddy/gasket/blob/main/packages/gasket-cli/bin/boot).